data-analytics/cloud-composer-examples/composer_dataflow_examples/README.md

```diff
@@ -6,17 +6,17 @@ The goal of this example is to provide a common pattern to automatically trigger
 
 ***
 
-<img src="img/workflow-overview.png" alt="Workflow Overview" height="400" width="800"/>
+![Alt text](../img/workflow-overview.png "A diagram illustrating the workflow we create in this example")
```

Comment:
@woogie-wolgemuth  this will be a paragraph of an alt-text to describe the whole architecutre diagram (also the text below this would identical content to this alt-text) would "A diagraming illustrating the workflow described below" be acceptable?